content/fundamentals/lifecycle-events.md

```diff
@@ -12,6 +12,8 @@ The following diagram depicts the sequence of key application lifecycle events,
 
 Lifecycle events happen during application bootstrapping and shutdown. Nest calls registered lifecycle hook methods on `modules`, `injectables` and `controllers` at each of the following lifecycle events (**shutdown hooks** need to be enabled first, as described [below](https://docs.nestjs.com/fundamentals/lifecycle-events#application-shutdown)). As shown in the diagram above, Nest also calls the appropriate underlying methods to begin listening for connections, and to stop listening for connections.
 
+> info **Info** Be aware that this page doesn't apply to [lazy loaded modules](https://docs.nestjs.com/fundamentals/lazy-loading-modules) as lifecycle hooks methods listed below won't get called on them.
```

Comment:
Instead of updating the lifecycle-events page, we should rather just add this hint in the lazy-loading-modules